vite.config.js

```diff
@@ -0,0 +1,14 @@
+import vue from '@vitejs/plugin-vue';
+import { defineConfig } from 'vite';
+
+const path = require('path');
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
```

Comment:
Should we add eslint and stylelint plugins? Does storybook displays linter errors in the overlay when using vite builder instead of webpack?

Reply:
It doesn't show. I'm not sure that overlay is helpful for DX. It sometimes is a little bit annoying, and all that overlay says - say, Eslint and Stylelint in a console. Are we sure that we need it?

Reply:
From my perspective is not a blocker to not have it but if I could have it in my environment I wouldn't turn it off.